applications/bci_visualization/streams/snirf.py

```
@@ -0,0 +1,156 @@
+
+
+class SNIRFStream(BaseNirsStream):
+    """
+    Streams data from a SNIRF file.
+    See more about the spec at https://github.com/fNIRS/snirf
+    """
+
+    def __init__(self, snirf_file: Path | str) -> None:
+        self._snirf_file_path = Path(snirf_file)
+        if not self._snirf_file_path.exists():
+            raise FileNotFoundError(f"SNIRF file '{snirf_file}' does not exist")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)